### PR TITLE
[Update] Japanese translation of docs/newbs_testing_debugging.md.

### PR DESCRIPTION
**Pull request by mtei**
## Description

Updated the Japanese translation of newbs_testing_debugging.md to 0.9.0.

```shell
git diff ed0575fc8 0.9.0 -- docs/newbs_testing_debugging.md
```

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [ ] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [x] Documentation

## Checklist

- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [ ] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

**Comment by mtei** [comment]
@umi-umi @shelaf @s-show @hsgw @satt99 @takashiski @yoshihara よかったらレビューお願いします。

File: docs/ja/newbs_testing_debugging.md [hunk 1]
```diff
@@ -34,6 +28,10 @@ void keyboard_post_init_user(void) {
 }
 ```
 
+## デバッグツール :id=debugging-tools
+
+キーボードのデバッグには2種類のツールがあります。
```
Comment (s-show) [inline docs/ja/newbs_testing_debugging.md:33]: 「デバッグには〜」を生かすと「2種類のツールが使えます」の方が自然な文章になるかなと思いますが、原文が「There are〜」となっていますので、「使えます」と訳すのは難しいと思います。なので、次のような訳はどうでしょうか。
```suggestion
キーボードのデバッグに使えるツールは2つあります。
```

File: docs/ja/newbs_testing_debugging.md [hunk 2]
```diff
@@ -2,25 +2,19 @@
...
 -->
 
 カスタムファームウェアをキーボードへ書き込んだら、テストする準備が整います。運が良ければ全て問題なく動作しているはずですが、もしそうでなければこのドキュメントがどこが悪いのか調べるのに役立ちます。
 
 ## テスト
 
-通常、キーボードをテストするのは非常に簡単です。全てのキーをひとつずつ押して、期待されるキーが送信されていることを確認します。キーを押したことを見逃さないためのプログラムもあります。
+通常、キーボードをテストするのは非常に簡単です。
+全てのキーをひとつずつ押して、期待されるキーが送信されていることを確認します。
+QMKを実行していなくても、[QMK Configurator](https://config.qmk.fm/#/test/)のテストモードを使ってキーボードを確認することができます。
```
Comment (shelaf) [inline docs/ja/newbs_testing_debugging.md:15]:
```suggestion
QMK を実行していなくても、[QMK Configurator](https://config.qmk.fm/#/test/) のテストモードを使ってキーボードを確認することができます。
```

File: docs/ja/newbs_testing_debugging.md [hunk 3]
```diff
@@ -2,25 +2,19 @@
...
-* [QMK Configurator](https://config.qmk.fm/#/test/) (Web Based)
-* [Switch Hitter](https://web.archive.org/web/20190413233743/https://elitekeyboards.com/switchhitter.php) (Windows Only)
-* [Keyboard Viewer](https://www.imore.com/how-use-keyboard-viewer-your-mac) (Mac Only)
-* [Keyboard Tester](http://www.keyboardtester.com) (Web Based)
-* [Keyboard Checker](http://keyboardchecker.com) (Web Based)
-
-## デバッグ
+## デバッグ :id=debugging
 
 `rules.mk`へ`CONSOLE_ENABLE = yes`の設定をするとキーボードはデバッグ情報を出力します。デフォルトの出力は非常に限られたものですが、デバッグモードをオンにすることでデバッグ情報の量を増やすことが出来ます。キーマップの`DEBUG`キーコードを使用するか、デバッグモードを有効にする [Command](ja/feature_command.md) 機能を使用するか、以下のコードをキーマップに追加します。
```
Comment (shelaf) [inline docs/ja/newbs_testing_debugging.md:19]: _summary.mdではカタカナになっていました
```suggestion
`rules.mk`へ`CONSOLE_ENABLE = yes`の設定をするとキーボードはデバッグ情報を出力します。デフォルトの出力は非常に限られたものですが、デバッグモードをオンにすることでデバッグ情報の量を増やすことが出来ます。キーマップの`DEBUG`キーコードを使用するか、デバッグモードを有効にする [コマンド](ja/feature_command.md) 機能を使用するか、以下のコードをキーマップに追加します。
```

**Pull request merged**